pkg/metric/writer_cw.go

```diff
@@ -31,12 +33,15 @@ const (
 	chunkSizeCloudWatch = 20
 	minusOneWeek        = -1 * 7 * 24 * time.Hour
 	plusOneHour         = 1 * time.Hour
+
+	defaultWriteTimeout = 10 * time.Second
```

Comment:
I am not sure I like the double default, we should be able to rely on the config parser setting the right defaults